### ISSUE TITLE
Doctor's tips/chat - Message log

### ISSUE THREAD

**Issue by andersand** [CorsixTH/CorsixTH#2033]
**What feature or enhancement would you like to add?**

TL;DR  ? skip first paragraph.

I have this "bad habit" you could call it, of clicking away the doctor automatically / by accident. Sometimes the stuff he talks about is actually useful though (cannot remember right now but there's a few I believe you don't want to miss, like some warning that you shouldn't ignore or you may have to revert to your last save). Now to the point (sorry)...

I sometimes wish there was a message log for doctor's comments, rants or what they're called. So if you miss a message you can click somewhere (or just a shortcut key) to open a dialog with recent messages of this kind.

**How would this feature benefit the gameplay of CorsixTH?**
Mentioned above.

**Was the feature in the original game?**
No

**What version of CorsixTH are you using?**
0.65.1

**Does this feature/enhancement relate to a current problem?**
Not to my knowledge.

**Additional information**

**Comment by lewri** [comment]
Do you mean the adviser messages? I think this has come up before. 
They occasionally tend to repeat. So it may be best if it was a system with context.

Then again you could just have back forward buttons of the last 12 messages.

**Comment by andersand** [comment]
That could work, as long as there is a way to "bring him up" like clicking somewhere on the toolbar. As far as I can see there is only one good way to do this without changing the original graphics. If you could bring up the message log in the text mode of the empty panel by right clicking it perhaps and then be able to switch back with another right click? The panel could for instance show the previous three advicer messages (just truncated to fit the width + ellipsis)
![image](https://user-images.githubusercontent.com/621686/128361884-93d772bb-f7a1-44f3-8bd7-ffd0bf5435bb.png)

**Comment by TheCycoONE** [comment]
Seems feasible, I also remember this coming up before.

**Comment by TheCycoONE** [comment]
Duplicate of #530 but I like your mock.
